python/fusion_engine_client/messages/defs.py

```diff
@@ -43,6 +43,7 @@ class MessageType(IntEnum):
   # INS solution messages.
   POSE = 10000
   GNSS_INFO = 10001
+  GNSS_SATELLITE = 10002
```

Comment:
would you be opposed to making this plural (since its a repeated message)

Reply:
Actually I take it back, this is more consistent with the class name and it'd be kind of weird to have it be Satellites in the class.